Pods/CreatubblesAPIClient/CreatubblesAPIClient/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,41 @@
+"error_invalid_response_detail" = "Response from the server is invalid.";
```

Comment:
where were these strings before?

Reply:
There were here from long time. It's a file located in `Pods` directory, so I guess that our demo app did not update pods from long time.